contrib/systemd/superset-webserver.service

```diff
@@ -0,0 +1,36 @@
+ #
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[Unit]
+Description =Superset Webserver Daemon
+After =network.target postgresql.service mysql.service redis.service rabbitmq-server.service
+Wants =postgresql.service mysql.service redis.service rabbitmq-server.service
+
+[Service]
+EnvironmentFile=/etc/sysconfig/superset
+User =superset
+Group =superset
+Type =simple
+ExecStart =/bin/gunicorn -w 10 -k gevent -t 120 -b  0.0.0.0:6666 --limit-request-line 0 --limit-request-field_size 0 --statsd-host localhost:8125 superset:app 
+Restart =on-failure
```

Comment:
Should this be more like: https://docs.gunicorn.org/en/stable/deploy.html#systemd

Reply:
Thanks.